src/web/components/KeyPairs/KeyPairsTable.scss

```diff
@@ -0,0 +1,49 @@
+.key-pairs {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+
+  .key-pairs-table {
+    margin-top: 28px;
+    font-size: 0.875rem;
+
+    thead {
+      font-weight: 700;
+      border: none;
+    }
+
+    th {
+      padding-right: 10px;
+
+      .action {
+        text-align: end;
+        padding-left: 140px;
+      }
+    }
+
+    td {
+      padding: 15px 5px;
```

Comment:
nit: I don't think we want padding on the left as well, so maybe:
```
padding: 15px 5px 15px 0;
```
Otherwise these get a bit out of line
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/cdfcafb0-4518-463a-8717-494c6437d7a3)

Reply:
fixed